packages/react-juce/src/lib/MacroProperties/Colors.ts

```diff
@@ -0,0 +1,251 @@
+import ColorString from "color-string";
+import ColorNames from "color-name";
+
+export default class Colors {
```

Comment:
A bit of a nitpick, but I don't think we need this to be a class, and in fact when we use this class with `let colors = new Colors()` we're making an object and then throwing it away almost immediately because we don't need the notion of a persistent colors object here.

What I think I'd prefer is to keep a const list of color properties at module scope:
```js
import ColorString from 'color-string';
import ColorNames from 'color-name';

const COLOR_PROPERTIES = [
    'border-color',
    'background-color',
    ...etc
];
```

Then we can just export a set of pure functions:

```
function isColorProperty(propKey: string) {
    return COLOR_PROPERTIES.includes(propKey);
}

...etc...

export default {
  isColorProperty,
  colorStringToAlphaHex,
  ...
}
```

Then when using it:

```
import Colors from 'lib/MacroProperties/Colors';

// No need to construct an object here
if (Colors.isColorProperty(blah)) {
    ...etc
}
```

Reply:
yup yup good call